pkg/scheduler/eventhandlers.go

```diff
@@ -134,6 +134,7 @@ func (sched *Scheduler) addPodToSchedulingQueue(obj interface{}) {
 func (sched *Scheduler) updatePodInSchedulingQueue(oldObj, newObj interface{}) {
 	start := time.Now()
 	defer metrics.EventHandlingLatency.WithLabelValues(framework.UnscheduledPodUpdate.Label).Observe(metrics.SinceInSeconds(start))
```

Comment:
wouldn't this be included as part of `PodSchedulingPropertieschange`?

Reply:
Good point. Sometimes Yes, but not always. I changed the implementation a bit so that we won't record `UnscheduledPodUpdate` event handling latency twice.